platform/engineering/official_github_teams_process

```diff
@@ -0,0 +1,38 @@
+# GitHub Teams on for VA.gov
+
+## What is GitHub Teams, and why are we talking about it?
+
+GitHub Teams give organizations the ability to create groups of members and control access to repositories. Team members can be granted read, write, or admin permissions to specific repositories.
+For more information on GitHub teams, please see [GitHub docs](https://help.github.com/en/enterprise/2.19/admin/user-management/creating-teams).
+
+At the moment, some teams have their own GitHub Teams, while others do not. GitHub Teams’ benefits and functions are not fully understood across VA.gov. Our goal is to provide a basic understanding of GitHub teams, establishing a logical structure of teams first. Then we can improve the process of granting access and developers’ experience on VA.gov. 
+
+In the future, we will be using *Official GitHub Teams* as a source of truth for team rosters, access, and auditing purposes for VSP. The process of creating and maintaining Official GitHub Teams is below. 
```

Comment:
We already have a set of these, right? They drive Jenkins permissions right now. Is the intent here to describe a future state past this or to describe what's in place now and how we're using them?

Reply:
Well, yes and no. The teams we have in place are only on VSP and even those are will change if the naming convention changes, so we would need to update the names to reflex the naming we go forward with.

Reply:
We have some of that, so clarify that in the text.

Reply:
taken out this starting section and will move into a Product outline